content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -0,0 +1,656 @@
+- =
+- !=
+
+### Example
+
+```YML
+field: enabled
+operator: =
+value: false
+```
+
+::: info
+If you want to select the activated products, set the value to "true". If you want to select the deactivated ones, set it to "false".
+:::
+
+## Completeness
+
+The possible operators for the `completeness` field are:  
+- =
+- !=
+- “>”
+- <
+
+#### Example
+
+```YML
+field: completeness
+locale: fr_FR
+scope: print
+operator: =
+value: "100"
+```
+
+::: warning
+The `locale` and `scope` elements are mandatory.
+:::
+
+::: info
+The `value` field requires a percentage.
+:::
+
+### Family
+
+The possible operators for the `family` field are:  
+- IN
+- NOT IN
+- EMPTY
+- NOT EMPTY
+
+::: warning
+If the operator is EMPTY or NOT EMPTY, the value element will be ignored.
+:::
+
+#### Example
+
+```YML
+field: family
+operator: IN
+value:
+ - camcorders
+ - digital_cameras
+```
+
+::: info
+The `field` expects the family code.
+:::
+
+
+### Groups
+
+The possible operators for the `groups` field are:  
+
+- IN
+- NOT IN
+- EMPTY
+- NOT EMPTY
+
+::: warning
+If the operator is EMPTY or NOT EMPTY, the value element will be ignored.
+:::
+
+### Example
+
+```YML
+field: groups
+operator: IN
+value:
+ - oro_tshirts
+ - akeneo_tshirts
+```
+
+::: info
+The `field` expects the group code.
+:::
+
+## Categories
+
+The possible operators for the `categories` field are:  
+
+- IN
+- NOT IN
+- UNCLASSIFIED
+- IN OR UNCLASSIFIED
+- IN CHILDREN
+- NOT IN CHILDREN
+
+### Example
+
+```YML
+field: categories
+operator: IN
+value:
+ - C0056
+ - F677
+```
+
+::: info
+The `field` expects the category code.
+:::
+
+# Attribute types
+
+Now that the actions and the fields don't have any secret for you anymore, you can discover which attribute types you can use as *targets* for your rules. It means that these attributes can receive the values you defined in your rules.
+
+## Text/Text area
+
+The possible operators for the `text/text area` attribute types are:  
+
+- STARTS WITH
+- ENDS WITH
+- CONTAINS
+- DOES NOT CONTAIN
+- =
+- !=
+- EMPTY
+- NOT EMPTY
+
+:::info
+If the operator is EMPTY or NOT EMPTY, the value element will be ignored.
+:::
+
+### Example
+
+```YML
+field: description
+operator: CONTAINS
+value: "Awesome product"
+```
+**text** can be written with or without quotation marks.
+
+## Metric
+
+The possible operators for the `metric` attribute type are:  
+- <
+- <=
+- =
+- !=
+- “>”
+- “>=”
+- EMPTY
+- NOT EMPTY
+
+:::info
+If the operator is EMPTY or NOT EMPTY, the value element will be ignored.
+:::
+
+### Example
+
+```YML
+field: weight
+operator: =
+value:
+ amount: 0.5
+ unit: KILOGRAM
+```
+
+We expect **numeric value** and **measurement unit code** in the following format:
+- dot “.” is the decimal separator,
+- there is no space between thousands.
+
+## Boolean
+
+The possible operators for the `boolean` attribute type are:  
+- >=
+- !=
+
+### Example
+
+```YML
+field: shippable_us
+operator: =
```

Comment:
The example doesn't match with the available operators

Reply:
wow thanks!!! I did a mistake in the operators list ;)